packages/expo-web-browser/ios/EXWebBrowser/EXWebBrowser.m

```diff
@@ -158,6 +163,38 @@ - (dispatch_queue_t)methodQueue
   }
 }
 
+UM_EXPORT_METHOD_AS(warmUpAsync,
```

Comment:
Could you also please update `expoModules` mocks? Run `apps/jest-expo-mock-generator` app on an iOS device, copy the logged blob into `jest-expo/…/expoModules.js` and prettify. 🙂